submissions/slack-status/src/background.js

```diff
@@ -0,0 +1,68 @@
+async function getSlackConfig() {
+    const xoxd = (await chrome.cookies.get({
+        name: "d",
+        url: "https://app.slack.com"
+    }))?.value;
+    if (!xoxd) throw new Error("Failed to get xoxd");
+
+    const slackTab = await chrome.tabs.create({
+        url: "https://app.slack.com/404",
+        active: false
```

Comment:
If it fails to grab the slack token it fails silently which is not great.

Reply:
Now opens a popup which asks you to sign in!